### PR TITLE
modal works on every breakpoint, need to add x button

### PR DESCRIPTION
**Pull request by sanchezfernandes**
-Almost done
-Need to get updated info from develop.

### CODE REVIEW

File: src/components/modal/Modal.jsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+const Modal = () => {
+
+    const [modal, setModal] = useState(false)
+
+    const toggleModal = () => {
+        setModal(!modal)
+    }
+
+    return (
+        <>
```
Comment (trumerfe) [inline src/components/modal/Modal.jsx:14]: Add name to top level container to make it semantic

**Review by Cob007 — APPROVED** [review]

File: src/components/modal/Modal.scss [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+    height: 100vh;
+    top: 0;
+    left: 0;
+    right: 0;
+    bottom: 0;
+    position: fixed;
+}
+
+.overlay {
+    // background: rgba(49, 49, 49, 0.8);
```
Comment (trumerfe) [inline src/components/modal/Modal.scss:16]: Try to not have commented code

**Review by trumerfe — APPROVED** [review]
Good code, just look @ comments in code

**Review by Cob007 — APPROVED** [review]

**Review by Cob007 — APPROVED** [review]

File: src/components/modal/Modal.jsx (hunk 1 above)
Comment (sanchezfernandes) [reply src/components/modal/Modal.jsx:14]: Got it, will do

File: src/components/modal/Modal.scss (hunk 2 above)
Comment (sanchezfernandes) [reply src/components/modal/Modal.scss:16]: Yes correct, not the final need to rem these px's.

**Review by Edthev — APPROVED** [review]
I agree with erik on his comments but otherwise looks good!

**Review by jwuzzle — APPROVED** [review]

**Pull request merged**